src/components/SingleDatePickerInputController.jsx

```diff
@@ -251,7 +255,9 @@ export default class SingleDatePickerInputController extends React.PureComponent
         small={small}
         regular={regular}
         verticalSpacing={verticalSpacing}
-      />
+      >
+        {children && children}
```

Comment:
@majapw I don't think so, because there wasn't a conditioner earlier. The case we're checking for here is whether **children** is `null`. In the case that **focused** is `true`, I still think that it's still possible to have **children** set to `null`